Source/relay/TourGuide/Quests/Level 11 - Copperhead.ash

```diff
@@ -308,8 +308,9 @@ void QLevel11ShenGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry
             if (__shen_start_day_to_assignments contains ++daycount)
                 subentry.entries.listAppend("Tomorrow will instead be:|*• " + __shen_start_day_to_assignments[daycount].shenAssignmentsJoinLocations().listJoinComponents("|*• "));
         }
-        if (my_daycount() == 1 && my_path().id != PATH_EXPLOSIONS)
-            subentry.entries.listAppend("Perhaps wait until tomorrow before starting this; day 2's shen bosses are more favourable.");
+        // This used to be true in old metas, but hasn't been true for a while. Commenting it out.
+        // if (my_daycount() == 1 && my_path().id != PATH_EXPLOSIONS)
+        //     subentry.entries.listAppend("Perhaps wait until tomorrow before starting this; day 2's shen bosses are more favourable.");
```

Comment:
Maybe just delete? Easy to get back from history if we need it later. Up to you though.